bgpd/bgp_main.c

```diff
@@ -380,6 +381,7 @@ int main(int argc, char **argv)
 		"  -p, --bgp_port     Set BGP listen port number (0 means do not listen).\n"
 		"  -l, --listenon     Listen on specified address (implies -n)\n"
 		"  -n, --no_kernel    Do not install route to kernel.\n"
+		"  -z, --no_zebra     Do not communicate with Zebra ( routes, vrf.\n"
```

Comment:
`-z` is already allocated by another option (`-z, --socket`). I think only `--no_zebra` should be enough here, no need to allocate a character for this option.

Also, please fix the help string (only `Do not communicate with zebra` would be enough I think).

Reply:
I will use '-Z' if you dont mind